README.md

```diff
@@ -6,19 +6,25 @@ about the species that was reported
 
 ## Technology stack
 
-- PostgreSQL 11
-- PostGIS 2.5.x
-- Python 3.11
-- Django 3.2 LTS
+- PostgreSQL 16
+- PostGIS 3.5.x
+- Python 3.12
+- Django 5.2 LTS
 - Google Maps
+- Bootstrap 5
 
 ## Getting started
 
 Ensure that you have Docker and Docker Compose installed in your host's environment.
 
 To use the provided Docker container definitions:
 
-    docker-compose up -d
+```bash
+docker compose build
```

Comment:
@quick007 compose up -d is probably preferrable here... why switch it to require build then up? If you are suggesting they force a rebuild we can just use the --build parameter for compose up.

Reply:
Fixed this